doc/qbk/07_concepts/BuffersGenerator.qbk

```diff
@@ -0,0 +1,128 @@
+[/
+    Copyright (c) 2016-2022 Vinnie Falco (vinnie dot falco at gmail dot com)
```

Comment:
Shouldn't your name be here

Reply:
if you edited the file your name should be on it. if you created the file from scratch then ONLY your name should be on it.

Reply:
Done